docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -132,17 +132,6 @@ To implement type inference for `object` properties, create a converter like the
 
 `Newtonsoft.Json` doesn't have a maximum depth limit by default. For <xref:System.Text.Json> there's a default limit  of 64, and it's configurable by setting <xref:System.Text.Json.JsonSerializerOptions.MaxDepth?displayProperty=nameWithType>.
 
-### Stack type handling
-
-In <xref:System.Text.Json>, the order of a stack's contents is reversed when it's serialized. This behavior applies to the following types and interface and user-defined types that derive from them:
```

Comment:
I like the wording here, so would be good to start with what we have in this section in the other doc.